presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -849,7 +849,7 @@ public static double betaCdf(
         return distribution.cumulativeProbability(value);
     }
 
-    @Description("Inverse of Cauchy cdf for a given probability, median, and scale (gamma)")
+    @Description("Inverse of Cauchy cdf for a given probability, median, and scale (gamma) parameters")
```

Comment:
I don't think "parameters" is needed at the end here

Reply:
Thanks @rschlussel - I've removed "parameters".